Gopkg.lock

```diff
@@ -80,14 +80,14 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:e6d1805ead5b8f2439808f76187f54042ed35ee26eb9ca63127259a0e612b119"
+  digest = "1:74ce8329385730bb2d01e4e91b66183beceeb1512084e4253653ac2439f1ef8c"
   name = "golang.org/x/sys"
   packages = [
     "windows",
     "windows/svc",
   ]
   pruneopts = ""
-  revision = "b4a75ba826a64a70990f11a225237acd6ef35c9f"
+  revision = "a34e9553db1e492c9a76e60db2296ae7e5fbb772"
```

Comment:
the mod files still have the "b4a7..." revision of golang.org/x/sys

Reply:
fixed